terraform/cross-device/client/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
```

Comment:
@grapesfrog Is this client application also available in the federated-compute repository?